bot/exts/evergreen/youtube.py

```diff
@@ -20,7 +20,7 @@
 
 
 @dataclass
-class Video:
+class SearchResult:
     """Represents one video."""
```

Comment:
```py
    """Represents one video."""
```
It doesn't really represent a video -- it represents a video search result. Besides the linting rules, I would leave without a docstring.